LUKeychainAccess/LUKeychainAccess.m

```diff
@@ -149,11 +149,21 @@ - (id)objectForKey:(NSString *)key {
   return object;
 }
 
+- (id)recursivelyFindObjectForKey:(NSString *)key fromBaseClass:(Class)cls {
+  id result = [self objectForKey:key ofClass:cls];
+
+  if (!result && [cls superclass] && [cls superclass] != NSObject.class && [cls superclass] != cls) {
```

Comment:
Is there actually an example of when `[cls superclass] == cls`? I'm blanking here